examples/nuxt/nuxt-headlesswp-gravity-forms/wp-env/README.md

```diff
@@ -0,0 +1,106 @@
+# Example: Gravity Forms in Headless WordPress with Nuxt/Vue
+
+This example shows you how to wire up a full headless WordPress backend—complete with Gravity Forms, WPGraphQL, and a pre-built "Questionnaire" form—alongside a Nuxt 3 front end that dynamically renders your forms using the GraphQL interfaces feature of WPGraphQL for Gravity Forms.
+
+## Features
+
+- **Headless WordPress with Gravity Forms**
+
+  - Stand up WP + WPGraphQL + Gravity Forms (and your Questionnaire JSON) in one command.
+
+- **Dynamic form rendering**
+
+  - Use GraphQL interfaces (`GfFieldWithLabelSetting`, `GfFieldWithChoicesSetting`, etc.) plus a single `useFormFields` composable to automatically map any new field type to the correct Vue component.
+
+- **Composable Nuxt 3 front end**
+
+  - Fetch your form schema with `useGravityForm` and render all fields via a dynamic `<component :is="resolveFieldComponent(field)" … />` pattern.
+
+- **Out-of-the-box data**
+  - Includes a SQL dump of WP + Gravity Forms data and a zipped uploads folder so you can dive right in.
+
+## Project Structure
+
+```
+├── components/           # Vue form-field components & barrel file
+├── composables/         # useGravityForm.js & useFormFields.js
+├── pages/              # Nuxt page (index.vue) that renders the form
+├── wp-env/
+│   ├── db/
+│   │   └── database.sql # WordPress + Gravity Forms schema & data
+│   ├── setup/
+│   │   └── .htaccess   # CORS + pretty-permalinks for wp-env
+│   └── uploads.zip     # wp-content/uploads media files
+├── .wp-env.json        # @wordpress/env configuration
+├── package.json        # wp-env + Nuxt dev scripts
+└── nuxt.config.ts      # Nuxt application config
+```
+
+## Running the Example with wp-env
+
+### Prerequisites
+
+- Node.js (v18+ recommended)
+- pnpm or npm/yarn
+- Docker (so that @wordpress/env can spin up the WP container)
+
+### Setup Repository and Install
+
+```bash
+git clone https://github.com/your-org/nuxt-gravityforms-example.git
+cd nuxt-gravityforms-example
+pnpm install
+
+echo "NUXT_PUBLIC_WORDPRESS_URL=http://localhost:8888" > .env
+```
+
+### Quick Start
+
+1. Unzip uploads, start WP, import DB, then launch Nuxt:
+
+   ```bash
+   pnpm example:build
+   ```
+
+2. Or run steps separately:
```

Comment:
@Fran-A-Dev I really like this. @moonmeister We should also consider this for other examples when we rewrite them for npm